sir_cake/seller/views.py

```diff
@@ -18,3 +21,12 @@ def new_product(request):
         form = NewItemForm()
 
     return render(request, 'seller/new_product.html', {'form': form})
+
+
+def item_detail(request, pk):
+    try:
+        item = Item.objects.get(id=pk)
+        item.assign_category_full_name()
```

Comment:
Will we need to call `assign_category_full_name` often? Should we rather call it in the `Item` constructor?